docs/spec/spec.md

```diff
@@ -0,0 +1,35 @@
+# Specification: Ballerina OS Library
+
+_Owners_: @daneshk @MadhukaHarith92  
+_Reviewers_: @daneshk  
+_Created_: 2021/11/10  
+_Updated_: 2021/11/10  
+_Issue_: [#2339](https://github.com/ballerina-platform/ballerina-standard-library/issues/2339)
+
+# Introduction
+The OS library is used to retrieve information about the operating system and its current users. It is part of Ballerina Standard Library. [Ballerina programming language](https://ballerina.io/) is an open-source programming language for the cloud that makes it easier to use, combine, and create network services.
```

Comment:
```suggestion
The OS Ballerina Standard Library is used to retrieve information about the operating system and its current users. It is programmed in the [Ballerina programming language](https://ballerina.io/), which is an open-source programming language for the cloud that makes it easier to use, combine, and create network services.
```

Reply:
Shouldn't it be "It is programmed in Ballerina programming language"?

Reply:
+1. Edited the comment.